### ISSUE TITLE
On DW and WM apps - font size on 'positive reinforcement'

### ISSUE THREAD

**Issue by 215MMS** [215MMS/FallsWCC#100]
The font size on the positive reinforcement messages (after 'race mode') is very large on DW and WM apps.

**Comment by philiprbrenan** [comment]
Agreed that the presentation of text needs to be worked on - it is not a trivial issue - but nor is it a show stopper.

**Comment by 215MMS** [comment]
Discussed and agreed with client that it's not urgent (jcb)
